introduction.md

```diff
@@ -5,3 +5,5 @@ This repository contains a list of dojo rules for the Deadly Vipers dojo
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?